tortoise/fields/base.py

```diff
@@ -206,10 +206,13 @@ def validate(self, value: Any):
         for v in self.validators:
             if self.null and value is None:
                 continue
-            elif isinstance(value, Enum):
-                v(value.value)
-            else:
-                v(value)
+            try:
+                if isinstance(value, Enum):
+                    v(value.value)
+                else:
+                    v(value)
+            except ValidationError as exc:
+                raise ValidationError(f"{self.model_field_name}: {exc}")
```

Comment:
Shall we use `raise ValidationError(f"{self.model_field_name}: {exc}") from exc`?

Reply:
str(exc) returns error's message, so we just upstream the text
any other way?

Reply:
I think `raise ... from None` would be better as traceback can be confusing